features/cache.feature

```diff
@@ -132,3 +132,20 @@ Feature: Managed the WordPress object cache
       """
       Error: Could not replace object 'bar' in group 'foo'. Does it not exist?
       """
+
+    When I run `wp cache clear-group add_multiple`
+    Then STDOUT should be:
+      """
+      Success: Cache group 'add_multiple' was flushed.
+      """
+
+    When I try `wp cache clear-group non_existing`
+    Then STDERR should be:
+      """
+      Error: Cache group 'non_existing' is not supported.
+      """
+    When I try `wp cache clear-group false_return`
+    Then STDERR should be:
+      """
+      Warning: Cache group 'false_return' was not be flushed.
+      """
```

Comment:
> I don't know how to create/fake a cache implementation for better behat testing.

@spacedmonkey Are there any cache implementations you're aware of that we could easily install?

Reply:
You could use this as upstream: https://github.com/rhubarbgroup/redis-cache/blob/develop/includes/object-cache.php

It will always have the latest Redis implementation ahead of releases.

Reply:
👍 

I might try to find a SQLite implementation so we don't have to add Redis to CI.

Reply:
Or APCu.

Reply:
https://github.com/OllieJones/sqlite-object-cache

Reply:
Thank's I'll take a look tomorrow or monday.

Reply:
https://github.com/OllieJones/sqlite-object-cache does not like WP-CLI....